politeiawww/politeiawww.go

```diff
@@ -156,11 +157,12 @@ func (p *politeiawww) handleVersion(w http.ResponseWriter, r *http.Request) {
 	log.Tracef("handleVersion")
 
 	versionReply := www.VersionReply{
-		Version: www.PoliteiaWWWAPIVersion,
-		Route:   www.PoliteiaWWWAPIRoute,
-		PubKey:  hex.EncodeToString(p.cfg.Identity.Key[:]),
-		TestNet: p.cfg.TestNet,
-		Mode:    p.cfg.Mode,
+		Version:           www.PoliteiaWWWAPIVersion,
+		Route:             www.PoliteiaWWWAPIRoute,
+		PubKey:            hex.EncodeToString(p.cfg.Identity.Key[:]),
+		TestNet:           p.cfg.TestNet,
+		Mode:              p.cfg.Mode,
+		TokenPrefixLength: pd.TokenPrefixLength,
```

Comment:
The politeiawww api should not be importing things from the politeiad api.  The correct way to do this would be to have a politeiad policy route that returns it, but we don't have that setup so just make this a `Policy` const in the www api.